Resources/Prototypes/Entities/Objects/Misc/spaceshroom.yml

```diff
@@ -0,0 +1,95 @@
+# Structure
+- type: entity
+  name: spaceshroom
+  parent: BaseStructure
+  id: Spaceshroom
+  description: A cluster of wild mushrooms that likes to grow in dark, moist environments.
+  components:
+  - type: Sprite
+    sprite: Objects/Misc/spaceshroom.rsi
+    state: structure
+  - type: Item
+    sprite: Objects/Misc/spaceshroom.rsi
+  - type: Transform
+    anchored: true
+  - type: InteractionOutline
+  - type: StaticPrice
+    price: 250
```

Comment:
this feels kinda high. idk if it actually is but just something to ponder.

Reply:
It's supposed to be kind of rare, both in-game and "lore-wise"

Reply:
Changed to 150